File: templates/python/instrument_meter_composition_stream.py
"""instrument_meter_composition_stream: 测量仪表组成流面积（stacked-area 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='instrument and metering: composition stream'):
    return make_template_figure('stacked_area', seed=2816, title=title, domain='instrument and metering', topic='composition stream')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/instrument_meter_composition_stream.m
function fig = instrument_meter_composition_stream()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('stacked_area', 2816, 'instrument and metering: composition stream', 'instrument and metering', 'composition stream');
end
